recipe/meta.yaml

```diff
@@ -99,14 +101,12 @@ outputs:
         - {{ cdt('mesa-libgl-devel') }}      # [linux]
         - python
         - dbus                               # [not win]
-        - qt 5.12.*
-        - enum34                             # [py27]
+        - qt 5.12.9
```

Comment:
```suggestion
        - qt
        - qt >=5.12.9
```

Reply:
So the one without the constraint is to pick up the cf-pinning, and then adding a constraint on top? OK sounds good.

Reply:
yes